docs/manual/other-topics/upgrade-to-v6.md

```diff
@@ -52,6 +52,50 @@ This method now tests for equality with [`_.isEqual`](https://lodash.com/docs/4.
 
 This method now throws `Sequelize.AggregateError` instead of `Bluebird.AggregateError`. All errors are now exposed as `errors` key.
 
+### Refactored hooks
+
+In order to streamline API:
+
+- All method style add hook functions have been removed in favor of a composition based approach.
+- Hook names have been removed, you can add and remove them by function reference instead which was supported before.
+
+This affects `Model`, `Sequelize` and `Transaction`.
+
+#### Composition
```

Comment:
Why are you calling these composition? :)

Reply:
Lack of a better word since the hooks the hooks are now a composition rather than a mix in, probably should call this `Examples` instead.